smart-contracts/scripts/all_networks.sh

```diff
@@ -7,7 +7,16 @@
 #
 
 ## all networks
-all_networks=("rinkeby" "polygon" "xdai" "mainnet" "bsc" "optimism")
+all_networks=(
```

Comment:
Should we get this from the package network? (not sure how doable that is tho!)

Reply:
It is more pratical to have it listed like this, s one can easily comment / add